nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -148,9 +164,28 @@ in {
 
     # PipeWire depends on DBUS but doesn't list it. Without this booting
     # into a terminal results in the service crashing with an error.
+    systemd.services.pipewire.bindsTo = [ "dbus.service" ];
```

Comment:
Is this something that could be upstreamed?

Reply:
Maybe. As it is now, I've just copied what was previously done for `systemd.user.pipewire`.

Reply:
Maybe @nglen would like to do that, as original author of this code and comment? :D

_References: https://github.com/NixOS/nixpkgs/commit/888c1fbf96c3c167d2216f4451438a335c1c547b, https://github.com/NixOS/nixpkgs/commit/f6745d06f5dc6c357e9bc101f524a5ced3562921_